packages/app/src/runner/event-manager.ts

```diff
@@ -393,7 +397,7 @@ export class EventManager {
 
           this.studioStore.initialize(config, runState)
 
-          const runnables = Cypress.runner.normalizeAll(runState.tests, hideCommandLog)
+          const runnables = Cypress.runner.normalizeAll(runState.tests, hideCommandLog, testFilter)
```

Comment:
Since this value is set in the specStore, any reason it's not directly pulled from here vs passing this around? `useSpecStore.testFilter`?